roles/ansible/tower/config-ansible-tower-ocp/tasks/setup_pvc.yml

```diff
@@ -0,0 +1,52 @@
+---
+- block:
+   - name: Ensure user is authenticated with OCP
+     shell: |
+       oc login {{ openshift_host }} \
+          --token {{ openshift_token }} \
+          --insecure-skip-tls-verify={{ openshift_skip_tls_verify | default(false) | bool }}
+     no_log: true
+
+   - name: Checking if target project exists on OCP..
+     command: oc get project {{ openshift_project }}
+     register: getProject
+     failed_when: false
+
+   - name: Creating target project..
```

Comment:
This may be a left-over, but there's one `oc get project` above and two `oc new-project` here - seems a bit redundant + not sure what this task actually does as it registers `getProject` over again. 

Another couple of comments:
- Probably doesn't make sense to have the project created inside of `setup_pvc` as that's a bit "hidden". Better idea would be to do so in the `main.yml` or a separate file for creating the project
-  What happens if the project already exists, but belongs to someone else? Probably will error out at some point. Is this something we should check for?

Reply:
You're right, that was overlooked and it has been fixed, I've also changed the file structure to be a bit more clean and self-explanatory. I will also look into handling the the condition in which project has already been created by other user.